### DISCUSSION TITLE
Next release's major features

### DISCUSSION THREAD

**Discussion by PierreZ** [foundationdb-rs/foundationdb-rs#23] [General]
I'm thinking about which big features should be included in the next versions of the crates. The latest release on crates.io is `0.5.0`.

We are now actively searching for faulty seeds in the bindingtester using scheduled Github Actions. So once the features has landed on main and correctness is green for several weeks, we can assume that the version is stable.

I can think of two possibilities:

### 0.5.1

* FDB 6.3 support validated with correctness

### 0.5.2

* FDB Directory layer validated with correctness

### 0.5.3

* FDB 7.0 support validated with correctness

### 0.6.0

* FDB 6.3 support validated with correctness
* FDB Directory layer validated with correctness

### 0.7.0

* FDB 7.0 support validated with correctness

## 

I like the second proposal, as it kind of follows FDB protocol version. This would also allow us to release a new major version (`0.6.0`) quite quickly, as the PR is already [opened but stuck for now](https://github.com/foundationdb-rs/foundationdb-rs/pull/19).

What do you think @yjh0502 @Speedy37 @bluejekyll? Any other proposals?

**Comment by bluejekyll** [comment]
I haven’t been watching fdb closely enough to provide useful info here. I guess one thing I’d be worried about with proposal 2 is really the path to 1.0. My sense is that might be a good goal for the 0.7 release.

**Reply by PierreZ** [reply]
You are right, I did not take into consideration reaching `1.0`. This should be the goal indeed. We can then add a third proposal that looks like this:

## Proposal 3

### 0.6.0

* FDB 6.3 support validated with correctness
* FDB Directory layer validated with correctness

### 1.0.0

* FDB 7.0 support validated with correctness

This will allow us to move towards `1.0`, while releasing a `0.6` with new features. What do you think?

**Reply by bluejekyll** [reply]
It sounds good to me :)

**Reply by PierreZ** [reply]
I'm voting for the third proposal 👍

**Comment by PierreZ** [comment]
I'm creating the milestones for both 0.6.0 and 1.0.0
